src/metabase/models/pulse.clj

```diff
@@ -293,7 +293,7 @@
   "Fetch a single *Pulse*, and hydrate it with a set of 'standard' hydrations; remove Alert columns, since this is a
   *Pulse* and they will all be unset."
   [pulse-or-id]
-  (some-> (t2/select-one Pulse :id (u/the-id pulse-or-id), :alert_condition nil)
+  (some-> (t2/select-one Pulse :id (u/the-id pulse-or-id))
```

Comment:
in principle we shouldn't use this function anymore because there is no longer `pulse`, we should use `retrieve-alert` instead. But the problem is that alert has `:card` instead of `:cards` like pulse does, so lots of rendering code will break if we change to `retrieve-alert`.

I plan to rework this on the 2nd milestone, where we remove pulse alltogether